src/completions.ts

```diff
@@ -296,28 +296,17 @@ export abstract class CompletionSourceFuse extends CompletionSource {
     }
 
     /** Call to replace the current display */
-    // TODO: optionContainer.replaceWith and optionContainer.remove don't work.
-    // I don't know why, but it means we can't replace the div in one go. Maybe
-    // an iframe thing.
     updateDisplay() {
-        /* const newContainer = html`<div>` */
-
-        while (this.optionContainer.hasChildNodes()) {
-            this.optionContainer.removeChild(this.optionContainer.lastChild)
-        }
+        const newContainer = this.optionContainer.cloneNode(false)
```

Comment:
`cloneNode()` always returns `Node`, so this needs a type assertion, otherwise we get an error on line 308:
```suggestion
        const newContainer = this.optionContainer.cloneNode(false) as HTMLElement
```